src/Draco.Compiler/Internal/Syntax/Formatting/Box.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Draco.Compiler.Internal.Syntax.Formatting;
+internal class Box<T>(T value)
```

Comment:
I don't necessarily understand the reason for this immutable box

Reply:
Because some properties only follow a value without ever being set.
That's the case for DoesReturnLine for example.